src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -133,11 +134,18 @@ export default class PathwayMapperTable extends React.Component<
 
         this._columns[IPathwayMapperTableColumnType.SCORE] = {
             name: 'Score',
-            render: (d: IPathwayMapperTable) => (
-                <span>
-                    <b>{d.score.toFixed(2)}</b>
-                </span>
-            ),
+            render: (d: IPathwayMapperTable) => {
```

Comment:
Let's just leave the default render as is. See my comments about an alternative solution.